src/scripts/main.js

```diff
@@ -1 +1,11 @@
 'use strict';
+
+const mainView = document.querySelector('#largeImg');
+const thumbsList = document.querySelector('#thumbs');
+
+thumbsList.addEventListener('click', (e) => {
+  const link = e.target.closest('.list-item__link');
+
+  e.preventDefault();
```

Comment:
```suggestion
thumbsList.addEventListener('click', (e) => {
  e.preventDefault();
  const link = e.target.closest('.list-item__link');
```

small advice - better define `e.preventDefault();` function at the beginning

Reply:
змінив, але тепер чомусь тести не проходить